src/ui.cpp

```diff
@@ -613,7 +613,12 @@ void uilist::show()
                 // to be used.
                 const auto entry = utf8_wrapper( ei == selected ? remove_color_tags( entries[ ei ].txt ) :
                                                  entries[ ei ].txt );
-                trim_and_print( window, point( pad_left + 4, estart + si ),
+                int x = pad_left + 4;
+                int y = estart + si;
+                entries[ei].drawn_info.text_x_start = x;
+                entries[ei].drawn_info.text_x_end = x + max_entry_len;
+                entries[ei].drawn_info.y = y;
+                trim_and_print( window, point( x, y ),
```

Comment:
Have you tested selecting an entry while filtering? Because if I understand the code correctly, filtered entries doesn't have their drawn info reset here, which might lead to incorrect return value when an entry is selected while filtering.

Reply:
no I didn't, I am going to fix this now

Reply:
ok, fixed already

Reply:
> Have you tested selecting an entry while filtering? Because if I understand the code correctly, filtered entries doesn't have their drawn info reset here, which might lead to incorrect return value when an entry is selected while filtering.

by my text, this method is keep being called, so actually the values are reset, but for fentries (filtered entries) only, so I edited find_entry_by_coordinate